packages/commerce-sdk/src/core/loyalty/index.ts

```diff
@@ -0,0 +1,213 @@
+import { SDKOptions, ThirdwebSDK } from "@thirdweb-dev/sdk";
+import crypto from "crypto";
+import { NextApiRequest } from "next";
+import { LoyaltyRewardsParams, ResponseBody, RewardTokensParams } from "../../../types";
+import { GET_ORDER_BY_ID_QUERY } from "../../lib/queries";
+import { getRawBody, shopifyFetchAdminAPI } from '../../lib/utils';
+
+/**
+ * Gift ERC20 loyalty tokens to a customer using their wallet address.
+ * 
+ * @description This function is used to gift loyalty tokens to a customer using their wallet address. The customer must have purchased and item from your store in order to receive the loyalty tokens. The function will check the order ID and verify that the customer has purchased an item from your store. If the customer has purchased an item from your store, the function will gift the customer the loyalty tokens.
+ * 
+ * @example
+ * ```
+ * const txHash = await rewardPoints({
+    nextApiRequest: {{next_api_request}},
```

Comment:
we should not make this specific to next instead we should take in the specific data we need